app/controllers/posts_controller.rb

```diff
@@ -1,43 +1,35 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @pagy, @posts = pagy(@user.posts)
+    @post = Post.where(author_id: params[:user_id])
+
+    render '/posts/index'
   end
```

Comment:
- [ ] You are yet to eliminate the N+1 query as suggested by the previous reviewer. Kindly do that.

  ```
  @posts = @user.posts.includes(:comments)
  ```